recipe/meta.yaml

```diff
@@ -28,7 +27,7 @@ test:
 
 about:
   home: 'http://www.python-excel.org/'
```

Comment:
home and doc_url should be https:
home: https://www.python-excel.org/
doc_url: https://xlrd.readthedocs.io/en/latest/